bcipy/tasks/rsvp/calibration/inter_sequence_feedback_calibration.py

```diff
@@ -81,37 +81,45 @@ def __init__(self, win, daq, parameters, file_save):
         self.nonletters = ['+', 'PLUS', 'calibration_trigger']
         self.valid_targets = set(self.alp)
 
-        self.feedback_buffer_time = self.parameters['feedback_buffer_time']
-        self.feedback_line_color = self.parameters['feedback_line_color']
         self.time_flash = self.parameters['time_flash']
 
-        # SSVEP response to letter flash estimate
-        self.stimulation_frequency = calculate_stimulation_freq(self.time_flash)
-
         self.downsample_rate = self.parameters['down_sampling_rate']
         self.filtered_sampling_rate = self.fs / self.downsample_rate
-        self.psd_method = PSD_TYPE.WELCH
-
-        # The channel used to calculate the SSVEP response to RSVP sequence.
-        self.psd_channel_index = self.PSD_CHANNEL_INDEX
+    
         self.device_name = self.daq.device_info.name
         self.channel_map = analysis_channels(self.daq.device_info.channels, self.device_name)
 
         # EDIT ME FOR FEEDBACK CONFIGURATION
+
+        self.feedback_buffer_time = self.parameters['feedback_buffer_time']
+        self.feedback_line_color = self.parameters['feedback_line_color']
+
+        self.psd_method = PSD_TYPE.WELCH
+
+        # The channel used to calculate the PSD from RSVP sequence.
+        self.psd_channel_index = self.PSD_CHANNEL_INDEX
+
         # filter parameters
         self.filter_low = self.parameters['filter_low_pass']
         self.filter_high = self.parameters['filter_high_pass']
-        self.fitler_order = self.parameters['filter_order']
+        self.filter_order = self.parameters['filter_order']
         self.notch_filter_frequency = self.parameters['notch_filter_frequency']
 
+        # get the feedback band of interest
+        self.psd_lower_limit = self.parameters['feedback_band_lower_limit']
+        self.psd_upper_limit = self.parameters['feedback_band_upper_limit']
+
         # psd band of interest to use for feeback (low, high)
-        self.psd_export_band = (
-            self.stimulation_frequency * .85,
-            self.stimulation_frequency * 1.15)
+        self.psd_export_band = (self.psd_lower_limit, self.psd_upper_limit)
 
         # length of time to use for PSD calculation
         self.trial_length = self.time_flash * self.stim_length
 
+        self.lvl_5_threshold = self.parameters['feedback_level_5_threshold']
+        self.lvl_4_threshold = self.parameters['feedback_level_4_threshold']
+        self.lvl_3_threshold = self.parameters['feedback_level_3_threshold']
+        self.lvl_2_threshold = self.parameters['feedback_level_2_threshold']
+
```

Comment:
I think it's probably clearer for the end user to have a separate parameter for each category, but if there would ever be a different number of feedback categories it might be more flexible to have a single parameter with a delimited list of values.